views/js/pciCreator/dev/textReaderInteraction/pciCreator.js

```diff
@@ -2,8 +2,9 @@
 define([
     './creator/widget/Widget',
     'taoQtiItem/qtiCreator/editor/customInteractionRegistry',
-    'tpl!./runtime/tpl/markup'
-], function (Widget, registry, markupTpl, handlebarsHelpers) {
+    'tpl!./runtime/tpl/markup',
+    'css!textReaderInteraction/runtime/css/textReaderInteraction'
+], function (Widget, registry, markupTpl) {
```

Comment:
This works but unfortunaltey it is not the ideal solution because the css will be included twice when the item is realoaded in the authoring. All required css files are indeed added to the item (first time) as illustrated in the following sequence diagram http://forge.taotesting.com/projects/tao/wiki/TAO_PCI_and_PIC#Under-the-hood, then it is included a second time  here with your fix (second time).
The right solution is to find what wrong is happening in the creation of an PCI.